guide/designing-products/user-research-resources.md

```diff
@@ -37,29 +37,53 @@ Below is a collection of published user research findings around Bitcoin, from a
 ## Research papers
 
 ###### [Non-adoption of crypto-assets: Exploring the role of trust, self-efficacy, and risk](https://voskart.de/pdf/crypto_adoption.pdf)
+
+*Over the last years, crypto-assets have gained significant interest from private investors, academia, and industry. While the user population and their motivations, perceptions, and behaviors have been studied, non-adopters and factors influencing their decision have been left unexplored. This work fills this knowledge gap and sheds light on the effects of trust, perceived self-efficacy, and risk, which have been shown to be the key antecedents to technology acceptance, on the adoption intention of non-users.*
+
 Published in May 2021 by Artemij Voskobojnikov, Svetlana Abramova, Konstantin Beznosov, and Rainer Böhme.
 
 ###### [Bits Under the Mattress: Understanding Different Risk Perceptions and Security Behaviors of Crypto-Asset Users](https://voskart.de/pdf/bits_under_mattress.pdf)
+
+*Prior empirical work has attempted to study end-user security from both technical and organizational perspectives. However, the link between individuals’ risk perceptions and security behavior was often obscured by the heterogeneity of the subjects in small samples. This paper contributes quantitative results from a survey of 395 crypto-asset users recruited by a novel combination of deep and broad sampling. The analysis accounts for heterogeneity with a new typology that partitions the sample in three robust clusters – cypherpunks, hodlers, and rookies – using five psychometric constructs.*
```

Comment:
```suggestion
*A survey-based exploration of different security behaviors among three three Bitcoin user archetypes: Cypherpunks, hodlers, and rookies. This study demonstrates that cypherpunks tend to use self-managed security solutions, whereas hodlers and rookies use either custodial solutions or non-custodial wallets. *
```
What do you think about a two-sentence format written in more plain English? First sentence describes the study and methods, second sentence briefly states the findings. Simple, but give enough information to understand what the study is about.